pkgs/development/python-modules/import-expression/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytest,
+  pytest-cov,
+  astunparse,
+}:
+buildPythonPackage rec {
+  pname = "import-expression";
+  version = "1.1.4";
+  src = fetchFromGitHub {
+    owner = "ioistired";
+    repo = "import-expression-parser";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-mll2NePB7fthzltLOk6D9BgaDpH6GaW4psqcGun/0qM=";
+  };
+  pythonImportsCheck = [
+    "import_expression"
+    "import_expression._codec"
+  ];
+  propagatedBuildInputs = [ astunparse ];
+  nativeCheckInputs = [ pytest pytest-cov ];
+  pytestFlagsArray = [
+    "tests.py"
+    "--cov import_expression"
+    "--cov-config .coveragerc"
+    "--cov-report term-missing"
+    "--cov-report html"
+    "--cov-branch"
+  ];
+
+  # Once https://github.com/ioistired/import-expression-parser/pull/17 gets
+  # merged, remove this.
+  prePatch = ''
+    substituteInPlace tests.py --replace "py.test" "pytest"
+  '';
+
+  meta = {
+    description = "Transpiles a superset of python to allow easy inline imports";
+    homepage = "https://github.com/ioistired/import-expression-parser";
+    license = with lib.licenses; [ mit psfl ];
```

Comment:
Where is the psfl license defined?

Reply:
In the repository:
https://github.com/NixOS/nixpkgs/blob/86ef6bd96b6279e1a4a53236d341f5df1ede3803/lib/licenses.nix#L1001